config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -18,7 +18,7 @@ presubmits:
         - make
         - test-style
     annotations:
-      testgrid-dashboards: sig-auth-secrets-store-csi-driver
+      testgrid-dashboards: sig-auth-secrets-store-csi-driver, sig-auth-secrets-store-csi-driver-presubmit
```

Comment:
> Removes sig-auth-secrets-store-csi-driver from dashboard group

Did you want to remove it?

Reply:
I updated the PR description. It's not possible to remove it. All the dashboards need to part of the `dashboard_group`

Reply:
```
--- FAIL: TestConfig (0.12s)
    config_test.go:340: Dashboard sig-auth-secrets-store-csi-driver should be in dashboard_group sig-auth
```
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/22162/pull-test-infra-bazel/1392579412947374080

Reply:
Gotcha. We do want to see status of all the jobs. With this PR we will see them in groups based on their types and one with all the jobs in it.

Reply:
That's right!